src/app/store.js

```diff
@@ -3,7 +3,15 @@ import { createStore } from 'redux'
 
 import { middleware } from './middleware'
 import { reducer } from './reducer'
+import { saveState } from './additional/additionalFunc'
 
 export const store = createStore(reducer, middleware)
 
+// Saving the store changing to the Local Storage
+store.subscribe(() => {
```

Comment:
nice solution. 

My idea where to hydrate/ryhydrate the store with a solution outthere. But i such a simple usecase its a very nice solution with no overhead.